### ISSUE TITLE
from here till determineType() could be on the interface

### ISSUE THREAD

**Issue by Maortz** [TotalyAwesomeCoder/JsonParser#7]
https://github.com/TotalyAwesomeCoder/JsonParser/blob/a5091c0e8478a8a1b96a45cac217e77a4478b357/Code/JsmnDataHandler.cpp#L8

**Comment by noamgalili1** [comment]
i dont think so because it is related to the jsmn implementation.. the interface supposed to work with all framework we might choose to use in the future..
